components/Events/Filters.js

```diff
@@ -0,0 +1,204 @@
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, React } from 'react';
+import events from '../../data/event';
+import styles from '../../styles/components/Events/Filters.module.scss';
+
+export default function filters(props) {
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+  const [chosenComms, setChosenComms] = useState({
+    studio: false,
+    icpc: false,
+    design: false,
+    cyber: false,
+    teachLA: false,
```

Comment:
We can do this to prevent having to do a workaround like L75-L76 of this file
```suggestion
    "teach-la": false,
```